src/utils/src/backend-supports/project-supports.ts

```diff
@@ -26,4 +37,4 @@ const projectDurationBuilder = (wps: any) => {
   return Math.round(durationWeeks);
 };
 
-export { projectDurationBuilder, calculateEndDate };
+export { calculateDuration as projectDurationBuilder, calculateEndDate };
```

Comment:
the refactoring just exported this as what it was before so it didn't really do anything. look into refactoring with VSCode so it refactors the whole thing